catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/TransformVisitor.java

```diff
@@ -45,6 +47,33 @@
  * in a future version of the library.</i>
  */
 public class TransformVisitor<T> extends AbstractFilterVisitor2 {
+  private static final Map<String, Consumer<FlatFilterBuilder<?>>> OPERATORS =
+      ImmutableMap.<String, Consumer<FlatFilterBuilder<?>>>builder()
+          .put("AND", FlatFilterBuilder::and)
+          .put("And", FlatFilterBuilder::and)
+          .put("OR", FlatFilterBuilder::or)
+          .put("Or", FlatFilterBuilder::or)
+          .put("AFTER", FlatFilterBuilder::after)
+          .put("After", FlatFilterBuilder::after)
+          .put("BEFORE", FlatFilterBuilder::before)
+          .put("Before", FlatFilterBuilder::before)
+          .put("ILIKE", b -> b.like(false, "", "", ""))
+          .put("PropertyIsLike", b -> b.like(false, "", "", ""))
+          //          .put("LIKE", b -> b.like(true, "", "", ""))
```

Comment:
❓ I wonder if we should maintain defaults for the 3 string arguments of the `PropertyIsLike` type. Not sure the caller would _ever_ override them and they strike me as implementation details. Thoughts?

Reply:
For now will keep the builder as pure as possible but open to changing this in the future.